rstudio.tera

```diff
@@ -0,0 +1,812 @@
+---
+whiskers:
+  version: "2.2.0"
+  matrix:
+    - flavor
+  filename: "catppuccin-{{ flavor.identifier }}.rstheme"
+---
+
+{%- set surface1_80 = surface1 | mod(opacity=0.8) -%}
+{%- set surface1_32 = surface1 | mod(opacity=0.32) -%}
+
+/* rs-theme-name: catppuccin-{{ flavor.identifier }} */
+/* rs-theme-is-dark: {% if flavor.dark %}TRUE{% else %}FALSE{% endif %} */
+
+:root {
+{%- for _, color in flavor.colors %}
+  --{{ color.identifier }}: #{{ color.hex }};
+{%- endfor %}
+}
+
+.ace_gutter {
+  background: var(--base);
+  color: var(--subtext0); /* other theme sets this to #838995, but this is not part of Nord palette */
+}
+
+.ace_gutter-active-line {
+  background: var(--surface1); /* other theme sets transparency to 0.32 */
+}
+
+.ace_print-margin {
+  width: 1px;
+  background: var(--surface0); /* other theme sets this to #e8e8e8, but this is not part of Nord palette */
+}
+
+.ace_editor, .rstudio-themes-flat.ace_editor_theme .profvis-flamegraph, .rstudio-themes-flat.ace_editor_theme, .rstudio-themes-flat .ace_editor_theme {
+  background-color: var(--base);
+  color: var(--subtext0);
+}
+
+.ace_cursor {
+  color: var(--subtext0);
+}
+
+.ace_marker-layer .ace_selection {
+  background: {{ css_rgba(color=surface1_80) }};
+}
+
+
+.ace_selection.ace_start {
+  box-shadow: 0 0 3px 0px var(--base);
+  border-radius: 2px;
+}
+
+/* unclear function, could not find in documentation */
+.ace_marker-layer .ace_step {
+  background: var(--surface0); /* other theme sets this to rgb(198, 219, 174), but this is not part of Nord palette */
```

Comment:
Same comment as above.

Reply:
Hi @sgoudham So sorry for this huge delay in my response. The *other theme* and *Nord theme* references are typos because copy-paste code. It'll be changed in next version. @uncenter told me to add **Whiskers**, but I know nothing about that in this moment. Maybe you guys could tell me if its worth to add Whiskers instead of not to change the theme is in this moment.

Reply:
No worries about the delay in response, glad to see that you're still around! @AngelBerihuete 

As described in the PR description above, [catppuccin/whiskers](https://github.com/catppuccin/whiskers) is a custom port creation helper tool that makes it easier to create and maintain Catppuccin themes. I'd highly encourage you to read or skim our [RFC](https://github.com/catppuccin/community/blob/main/rfc/CAT-0003-Whiskers.md#introduction) to learn more about why this was created. 

As part of this PR, I was going to ask if you're still interested in maintaining the theme for RStudio (it's perfectly okay to say no if you're not that interested anymore!) and if you're okay with merging this PR to generate the theme files via whiskers.

Reply:
Hi @sgoudham I think is time to say good bye to this project and release it to the catppuccin community. I am not software engineer and my time is focused on Statistics research. So things like catppuccin/whiskers are far way from my interest.

So, please, if you know how to release the project to the community, I appreciate your help.

Reply:
No worries at all @AngelBerihuete, thank you for confirming with me and thank you again for your contribution to Catppuccin.

I will remove you from being the maintainer of this repository and merge this PR as it will better enable the community to contribute to this project. 

Best of luck with your statistics research!